src/components/editor/FormWrapper.jsx

```diff
@@ -13,23 +15,28 @@ class FormWrapper extends Component{
     } else {
       return (
         <form style={dashedBorder}>
-          <div className='FormWrapper'>
+          <div className='FormWrapper form-group'>
             <p>BEGIN FormWrapper</p>
               <div>
-                {this.props.propertyTemplates[0].map(function(pt, index){
+                {this.props.propertyTemplates.map(function(propTemplate, index){
+
+                  let pt = propTemplate[0]
+
                   if(pt.type == 'literal'){
                     return(
                       <InputLiteral propertyTemplate = {pt} key = {index} />
                     )
                   }
-                  else if (pt.type == 'resource'){
+                  if (pt.type == 'resource'){
                     return (<p key={index}><b>{pt.propertyLabel}</b>: <i>resource</i> type</p>)
                   }
-                  else if (pt.type == 'lookup'){
-                    return (<p key={index}><b>{pt.propertyLabel}</b>: <i>lookup</i> type</p>)
+                  if (pt.type == 'lookup'){
+                    return(
+                      <InputLookup propertyTemplate = {pt} key = {index} />
+                    )
                   }
-                  else if (pt.type == 'target') {
-                    return (<p key={index}><b>{pt.propertyLabel}</b>: <i>target</i> type</p>)
+                  if (pt.type == 'target') {
+                    return (<p key={index}><b>{pt.propertyLabel}</b>: <i>lookup</i> type</p>)
```

Comment:
I don't think we want this to say "lookup" -- but instead keep it as "target", yes?

Reply:
oops